cmd/flags/force.go

```diff
@@ -19,7 +19,7 @@ func DoYouConfirm() (bool, error) {
 	}
 	fmt.Println() //nolint:forbidigo
 
-	answer = strings.ToLower(strings.Trim(answer, " \n\t"))
+	answer = strings.ToLower(strings.Trim(answer, " \r\n\t"))
```

Comment:
Does this cause any bother on linux/macos? I wonder if there's go constant that evaluates to the RightThing(tm) for the platform

Reply:
Actually ignore me, I see that's a set of characters to trim not something you're adding :)